distribution/docs/src/main/resources/content/_securing/auditing.adoc

```diff
@@ -69,8 +69,8 @@ If you want to change the location of your systems security backup log from the
 [source,xml,linenums]
 ----
 <RollingFile name="securityBackup" append="true" ignoreExceptions="false"
-                     fileName="${sys:karaf.data}/log/<NEW_LOG_FILE>"
-                     filePattern="${sys:karaf.data}/log/<NEW_LOG_FILE>-%d{yyyy-MM-dd-HH}-%i.log.gz">
```

Comment:
I updated these lines because the purpose of this section of the documentation is to change the location of the backup log files, not the name.